### ISSUE TITLE
Cannot merge conflicts anymore

### ISSUE THREAD

**Issue by igorkulman** [Lightricks/Kintsugi#59]
The tool stopped working for me, if I point it to a project with conflicts I just get this, an error saying there is a conflict.

```
❯ kintsugi apps/macos/macOS.xcodeproj/project.pbxproj
Traceback (most recent call last):
	7: from /Users/igorkulman/.rbenv/versions/2.7.2/bin/kintsugi:23:in `<main>'
	6: from /Users/igorkulman/.rbenv/versions/2.7.2/bin/kintsugi:23:in `load'
	5: from /Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/kintsugi-0.5.1/bin/kintsugi:34:in `<top (required)>'
	4: from /Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/kintsugi-0.5.1/lib/kintsugi/cli.rb:195:in `block in create_root_command'
	3: from /Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/kintsugi-0.5.1/lib/kintsugi.rb:35:in `resolve_conflicts'
	2: from /Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/xcodeproj-1.21.0/lib/xcodeproj/project.rb:113:in `open'
	1: from /Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/xcodeproj-1.21.0/lib/xcodeproj/project.rb:211:in `initialize_from_file'
/Users/igorkulman/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/xcodeproj-1.21.0/lib/xcodeproj/plist.rb:21:in `read_from_path': \e[31m[!] The file `apps/macos/macOS.xcodeproj/project.pbxproj` is in a merge conflict. (Xcodeproj::Informative)
\e[0m
```

Using 0.5.1.

**Comment by byohay** [comment]
Hey @igorkulman, it seems that I have accidentally pushed uncommitted changes to the repository 🤦 Will fix it soon, and I'll also work on preventing it from happening again.

**Comment by byohay** [comment]
@igorkulman fixed in version 0.5.2.
